Update Automation Pricing with Examples (#1036)
smartcontractkit/documentation: docs/chainlink-automation/automation-economics.md

Chainlink Automation only requires an execution fee for transactions on-chain. This fee includes the transaction cost, a percentage premium (refer to the formula below), and a small fixed gas overhead accounting for gas between the network and the registry. The premium compensates the Automation Network for monitoring and performing your upkeep. The premium varies by chain and is listed on our [Supported Networks](../supported-networks/) page.

![Automation Pricing Formula](/images/contract-devs/automation/automation-pricing.png)

There is no registration fee or other fees for any off-chain computation.

**Example**: An upkeep transaction was executed on *Polygon mainnet*. It used *1.16M* gas at a gas price of *183.3 gwei*. The premium percentage on Polygon is *70%* (as of October 2022) and the Matic/LINK exchange rate is *0.0131*. The upkeep's LINK balance will be reduced by a fee of *0.0495 LINK*.

![Automation Pricing Example](/images/contract-devs/automation/automation-pricing-example.png)

## How Funding Works

Upkeeps have a LINK (ERC-677) balance. Every time an on-chain transaction is performed for your upkeep, its LINK balance will be reduced by the LINK fee. 

> Your upkeep's balance must exceed the minimum balance. If this requirement is nto met, the Automation Network will not perform on-chain transactions. You can add funds using the [Chainlink Automation App](https://automation.chain.link/) or by directly calling the `addFunds()` function on the `AutomationRegistry` contract. Anyone can call the `addFunds()` function.

Reviewer: ```suggestion
Your upkeep's balance must exceed the minimum balance. If this requirement is not met, the Automation Network will not perform on-chain transactions. You can add funds using the [Chainlink Automation App](https://automation.chain.link/) or by directly calling the `addFunds()` function on the `AutomationRegistry` contract. Anyone can call the `addFunds()` function.
```

## Withdrawing funds

To withdraw a LINK balance, you must cancel your upkeep first. Any upkeep that has not spent more than an aggregated amount of 0.1 LINK fees over the span of its lifetime is subject to a *0.1 LINK* fee. This cancellation fee protects node operators from spammers who register jobs that never perform.

> **Example 1**: My upkeep has spent *4.8 LINK* over its lifetime and has a balance of *5 LINK*. When it is cancelled, I will receive *5 LINK*.
> **Example 2**: My upkeep has spent *0 LINK* over its lifetime and has a balance of *5 LINK*. When it is cancelled, I will receive *4.9 LINK*.

Reviewer: ```suggestion
**Example 1**: Your upkeep has spent *4.8 LINK* over its lifetime and has a balance of *5 LINK*. When it is cancelled, you receive *5 LINK*.
**Example 2**: Your upkeep has spent *0 LINK* over its lifetime and has a balance of *5 LINK*. When it is cancelled, you receive *4.9 LINK*.
```
